games-emulation/ppsspp-libretro/ppsspp-libretro-1.19.2-r1.ebuild

```diff
@@ -29,7 +30,7 @@ RDEPEND="
 	dev-libs/libzip:=
 	media-libs/libpng:=
 	media-libs/libsdl2[joystick]
-	system-ffmpeg? ( media-video/ffmpeg:0/58.60.60[pic] )
```

Comment:
pic use flag doesn't exist anymore, --enable-pic is passed by default in ffmpeg ebuild.